docs/src/contributing_websocket_http_api.md

```diff
@@ -1231,6 +1407,120 @@ Mods and admins can remove and lock a post, creators can delete it.
 
 `PUT /post`
 
+#### Delete Post
+##### Request
+```rust
+{
+  op: "DeletePost",
+  data: {
+    edit_id: i32,
+    deleted: bool,
+    auth: String,
+  }
+}
+```
+##### Response
+```rust
+{
+  op: "DeletePost",
+  data: {
+    post: PostView
+  }
+}
+```
+
+##### HTTP
+
+`POST /post/delete`
+
+#### Remove Post
+
+Only admins and mods can remove a post.
+
+##### Request
+```rust
+{
+  op: "RemovePost",
+  data: {
+    edit_id: i32,
```

Comment:
Shouldnt this be called `post_id`?

Reply:
I just kept all these as `edit_id`, I spose I *could* rename all of them to `post_id`, `community_id`, `private_message_id`, etc in both the front and back end, but I think its pry not necessary because all the endpoints are named by what you are editing: `MarkCommentAsRead`, etc.

Reply:
Its not that important, but seems a bit confusing to me.